oscarlab/graphene
# [Examples] Add SQLite

As requested by @dimakuv. This is on top of #2481 (fcntl locks), because it fails without these.

## Description of the changes 

A very simple example for `sqlite`. Wraps the Ubuntu executable.

## How to test this PR? 

There is a simple sanity check (`make regression`).


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/oscarlab/graphene/2493)
